### DISCUSSION TITLE
Change Size of LoRA

### DISCUSSION THREAD

**Discussion by samuelkurt** [bmaltais/kohya_ss#2573] [Q&A] [answered]
I'm creating a LoRA and the file size is around 10MB. It doesn't matter how many pictures I use, how many steps, or how large the Network Alpha is. Can anyone tell me how to increase the size? I've seen LoRAs as big as 500MB.

**Comment by b-fission** [comment, marked as answer]
Network Rank is what you're looking for. The value of Alpha is related to learning rate.

**Reply by samuelkurt** [reply]
Thanks, that fixed it.
